internal/utils/proto.go

```diff
@@ -43,8 +44,10 @@ func UnmarshalCredentials(credentialsBase64 string) (*protos.Credentials, error)
 
 // returns enclave_id as hex-encoded string of SHA256 hash over enclave_vk.
 func GetEnclaveId(attestedData *protos.AttestedData) string {
+	// hash enclave vk
 	h := sha256.Sum256(attestedData.EnclaveVk)
-	return hex.EncodeToString(h[:])
+	// encode and normalize
+	return strings.ToLower(hex.EncodeToString(h[:]))
```

Comment:
`ecc/chaincode/enclave/mock_enclave.go` has also one of these which you will have to change (but as mentioned above, for consistency probably better to leave this here as-is and change c-code to `toupper`?

Reply:
ported both to toupper